examples/service-clients/odsp-client/shared-tree-demo/.eslintrc.js

```diff
@@ -0,0 +1,9 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+module.exports = {
+	extends: [require.resolve("@fluidframework/eslint-config-fluid/minimal"), "prettier"],
```

Comment:
```suggestion
	extends: [require.resolve("@fluidframework/eslint-config-fluid/strict"), "prettier"],
```

If you can't use `strict` for some reason, at least use `recommended`.